docusign/docusign.lua

```diff
@@ -0,0 +1,89 @@
+--[[
+# [DocuSign API](https://docs.docusign.com/esign/guide/usage/quickstart.html) connector.
+
+### Authentication
+
+Authentication use [Authentication](https://docs.docusign.com/esign/guide/authentication/legacy_auth.html).
```

Comment:
I gave you this anchor already, [DocuSign legacy header authentication] should be in a square braces